drivers/gles3/storage/texture_storage.cpp

```diff
@@ -748,6 +753,27 @@ void TextureStorage::texture_free(RID p_texture) {
 	texture_owner.free(p_texture);
 }
 
+RID TextureStorage::texture_set_external(RID p_texture, int p_width, int p_height) {
```

Comment:
We've discussed this before, but if we're going to call this "external" (which I think makes sense because `GL_TEXTURE_EXTERNAL_OES`) we need to rename `TextureStorage::texture_create_external()` to something else.

Reply:
agreed but still open to suggestions :)

Reply:
@dsnopek I'm going to open a proposal and clean the code so that it can be reviewed. What would be a good replacement for texture_set_external ? Something like texture_reallocate ?